espnet/nets/pytorch_backend/ctc.py

```diff
@@ -22,7 +22,10 @@ def __init__(self, odim, eprojs, dropout_rate, ctc_type='warpctc', reduce=True):
         self.dropout_rate = dropout_rate
         self.loss = None
         self.ctc_lo = torch.nn.Linear(eprojs, odim)
-        self.ctc_type = ctc_type
+
+        # In case of Pytorch >= 1.2.0, CTC will be always builtin
+        torch_ver = int(torch.__version__.replace('.', '').replace('post2', ''))
```

Comment:
It is better to use LooseVersion (see setup.py for example).
Also it is better to add logging.warning for notification if ctc_type is different.